spec/requests/discourse_automation_append_last_checked_by_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../discourse_automation_helper'
+
+describe DiscourseAutomation::AppendLastCheckedByController do
+  describe '#post_checked' do
+    fab!(:post) { Fabricate(:post) }
+    fab!(:topic) { post.topic }
+
+    it 'updates the topic custom fields' do
+      admin = Fabricate(:admin)
+      sign_in(admin)
+
+      put "/append-last-checked-by/#{post.id}.json"
+      expect(response.status).to eq(200)
+      expect(topic.custom_fields[DiscourseAutomation::TOPIC_LAST_CHECKED_BY]).to eq(admin.username)
+      expect(topic.custom_fields[DiscourseAutomation::TOPIC_LAST_CHECKED_AT]).to eq(Time.zone.now.to_s)
```

Comment:
When checking expectations on a date/time/timestamp, we use the `be_within_one_second_of` helper (or one of its variants) to ensure it's somewhat safe from flakyness